pkg/util/kernel/ip.go

```diff
@@ -0,0 +1,14 @@
+package kernel
```

Comment:
Just a drive-by comment but does this file need `// +build linux` preamble?

Reply:
Yes! Thanks for driving by. 😁